ipa-core/src/query/runner/hybrid.rs

```diff
@@ -165,7 +179,7 @@ mod tests {
         // TODO: When Encryption/Decryption exists for HybridReports
         // update these to use that, rather than generating OprfReports
         vec![
-            TestRawDataRecord {
+            /*TestRawDataRecord {
```

Comment:
delete?

Reply:
Currently the test only uses raw records for conversions (impressions are commented). Ideally we'd have both, but I'm not sure how exactly we want to structure HybridInfo to allow that. I mention this a bit in the initial PR message